Cargo.toml

```diff
@@ -33,13 +33,11 @@ regex = "~0.1.69"
 default     = ["suggestions", "color", "wrap_help"]
 suggestions = ["strsim"]
 color       = ["ansi_term", "libc"]
-yaml        = ["yaml-rust"]
 wrap_help   = ["libc", "term_size"]
-lints       = ["clippy", "nightly"]
-nightly     = [] # for building with nightly and unstable features
```

Comment:
I get the difference now, and also realize that removing a feature flag is a breaking.
Due to the feature name i assumed `clap_app!` required a nightly compiler.

I got a notification for the first comment, but I haven't had time to respond.